soc/silabs/silabs_siwx91x/siwg917/Kconfig.defconfig

```diff
@@ -9,4 +9,12 @@ config NUM_IRQS
 config BUILD_OUTPUT_HEX
 	default y
 
+config PM
```

Comment:
We shouldn't need to redefine these Kconfig symbols, they are defined in subsys/pm/.